assets/in

```diff
@@ -72,7 +73,15 @@ for branch in $fetch; do
   git branch $branch FETCH_HEAD
 done
 
+return_ref=""
+if [ -n "$tag_filter" ]; then
```

Comment:
In this PR we only check if `tag_filter` is defined to decide if we should report back a tag in the version or the git hash (Note that maybe this is not the ideal behaviour implementation as commented in https://github.com/concourse/git-resource/pull/46#issuecomment-208901148)

But the matching of `tag_filter` of happens in the `check` and it was implemented in #31